docs/advanced/document-store/qdrant.md

```diff
@@ -174,10 +177,10 @@ for embedding, price in zip(da.embeddings, da[:, 'tags__price']):
 ```
 
 Consider we want the nearest vectors to the embedding `[8. 8. 8.]`, with the restriction that
-prices must follow a filter. As an example, let's consider that retrieved documents must have `price` value lower
-or equal than `max_price`. We can encode this information in annlite using `filter = {'price': {'$lte': max_price}}`.
+prices must follow a filter. As an example, let's consider that retrieved Documents must have `price` value lower
```

Comment:
this does not have a fix proposed by @alexcg1

Reply:
If you are talking about the change from "documents" to "Documents", I think I have already changed that, sorry here, I have checked several times, but my version looks the same as the one Alex proposed.